mlir-sycl/test/Transforms/inliner.mlir

```diff
@@ -196,16 +211,19 @@ func.func private @inline_hint_callee() -> i32 attributes {passthrough = ["inlin
 
 func.func private @private_callee() -> i32 {
   %c_i32 = arith.constant 2 : i32
-  %res1 = sycl.call @callee_() {MangledFunctionName = @callee, TypeName = @A} : () -> i32
+  %res1 = sycl.call @main_() {MangledFunctionName = @main, TypeName = @A} : () -> i32
   %res2 = arith.addi %c_i32, %res1 : i32
   return %res2 : i32
 }
 
-func.func @callee() -> i32 {
-  %c1 = arith.constant 1 : i32
-  %c2 = arith.constant 2 : i32
+func.func private @get(%id: memref<?x!sycl_id_1_, 4>, %i: i32) -> i64 {
+  %c2_i64 = arith.constant 2 : i64
+  return %c2_i64 : i64
+}
+
+func.func @main(%id: memref<?x!sycl_id_1_>) -> (i32, i64) {
```

Comment:
Had to rename it to avoid conflicts with above function and avoid additional checks just to make `AGGRESSIVE` skip first bunch of functions.